renderer/components/Calendar/Calendar.tsx

```diff
@@ -47,48 +49,59 @@ type CalendarProps = {
   setSelectedDate: Dispatch<SetStateAction<Date>>;
```

Comment:
let's rewrite name of constant with uppercase
https://www.javascripttutorial.net/es6/javascript-const/